storage/include/rocksdb/transaction.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * transaction.h
+ *
+ *  Created on: 20 Aug 2019
+ *      Author: tkournik
+ */
+
+#include <rocksdb/utilities/transaction.h>
+#include "storage/db_interface.h"
+#include "client.h"
+#pragma once
+
+namespace concord {
+namespace storage {
+namespace rocksdb {
+
+
+#define ROCKSDB_THROW(action, status) \
+  throw std::runtime_error("rocksdb error: action: " action ", txn id[" + getIdStr() + std::string("], reason: ") + status.ToString())
+
+class Transaction: public ITransaction {
+ public:
+  Transaction(::rocksdb::Transaction* txn, ID id): ITransaction(id), txn_(txn){}
```

Comment:
txn? txn_? Don't be miserly, name it 'transaction'.

Reply:
:-)